src/diffusers/utils/testing_utils.py

```diff
@@ -627,10 +627,10 @@ def load_numpy(arry: Union[str, np.ndarray], local_path: Optional[str] = None) -
     return arry
 
 
-def load_pt(url: str, map_location: str):
+def load_pt(url: str, map_location: Optional[str] = None):
     response = requests.get(url, timeout=DIFFUSERS_REQUEST_TIMEOUT)
     response.raise_for_status()
-    arry = torch.load(BytesIO(response.content), map_location=map_location)
+    arry = torch.load(BytesIO(response.content), map_location=map_location, weights_only=False)
```

Comment:
1. `weights_only` default value changed from `False` to `True` from 2.6, so set explicitly to `False` here, otherwise `tests/pipelines/text_to_video_synthesis/test_text_to_video_zero.py::TextToVideoZeroPipelineSlowTests::test_full_model
` will raise error
2. set `map_location` default as None which aligns w. `torch.load`, otherwise  `tests/pipelines/text_to_video_synthesis/test_text_to_video_zero.py::TextToVideoZeroPipelineSlowTests::test_full_model
` will raise error

Reply:
Could we add `weights_only=True` as an arg to `load_pt` and pass in to torch load. In the test itself we can set it to false so that it's clear what's happening in the test.   I would avoid doing this under the hood because it is a potential security hole.

Reply:
@DN6 , pls help review, thx.